src/user/user.controller.ts

```diff
@@ -1,34 +1,74 @@
-import { Controller, Get, Post, Body, Patch, Param, Delete } from '@nestjs/common';
-import { UserService } from './user.service';
-import { CreateUserDto } from './dto/create-user.dto';
-import { UpdateUserDto } from './dto/update-user.dto';
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Patch,
+  Param,
+  Delete,
+} from '@nestjs/common';
+import { UserService } from '@app/user/user.service';
+import { CreateUserDto } from '@app/user/dto/create-user.dto';
+import { UpdateUserDto } from '@app/user/dto/update-user.dto';
 
+/**
+ * whatever the string pass in controller decorator it will be appended to
+ * API URL. to call any API from this controller you need to add prefix which is
+ * passed in controller decorator.
+ * in our case our base URL is http://localhost:3000/user
+ */
 @Controller('user')
 export class UserController {
   constructor(private readonly userService: UserService) {}
 
+  /**
+   * Post decorator represents method of request as we have used post decorator the method
+   * of this API will be post.
+   * so the API URL to create User will be
+   * POST http://localhost:3000/user
+   */
   @Post()
   create(@Body() createUserDto: CreateUserDto) {
-    return this.userService.create(createUserDto);
+    return this.userService.createUser(createUserDto);
   }
 
+  /**
+   * we have used get decorator to get all the user's list
+   * so the API URL will be
+   * GET http://localhost:3000/user
+   */
   @Get()
   findAll() {
-    return this.userService.findAll();
+    return this.userService.findAllUser();
   }
 
+  /**
+   * we have used get decorator with id param to get id from request
+   * so the API URL will be
+   * GET http://localhost:3000/user/:id
+   */
   @Get(':id')
```

Comment:
change ':id' parameter to 'id/:id',because if add one more @Get() with ':id' you will have a conflict